internal/optimizer/imagepullpolicyoptimizer_test.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright IBM Corporation 2020
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package optimize
+
+import (
+	"github.com/google/go-cmp/cmp"
+	"github.com/konveyor/move2kube/internal/types"
+	plantypes "github.com/konveyor/move2kube/types/plan"
+	log "github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	"testing"
+)
+
+func TestImagePullPolicyOptimizer(t *testing.T) {
+	log.SetLevel(log.DebugLevel)
+
+	t.Run("IR with no services", func(t *testing.T) {
+		// Setup
+		p := plantypes.NewPlan()
+		ir := types.NewIR(p)
+		imagePullPolicyOptimizer := imagePullPolicyOptimizer{}
+		want := ir
```

Comment:
In this case, can  we ignore `want` variable and replace tests with `if !cmp.Equal(actual, ir){}` ?

Reply:
@ShajithaMohammed in this case it might not make a difference but in general that would run into the same issue.
Can you please keep the expected output and actual output separate to avoid having to think about whether they are affecting each other?